README.md

```diff
@@ -1,6 +1,8 @@
 Iris
 ====
 
+[![Join the chat at https://gitter.im/SciTools/iris](https://badges.gitter.im/SciTools/iris.svg)](https://gitter.im/SciTools/iris?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
```

Comment:
I guess it's that blank line on line number 5 which is separating the badge onto it's own line. 
I'm not sure how to modify this PR so I will close this one and open another one with my own branch